gax-java/gax-grpc/src/main/java/com/google/api/gax/grpc/ChannelPool.java

```diff
@@ -542,7 +548,14 @@ public void onClose(Status status, Metadata trailers) {
       } catch (Exception e) {
         // In case start failed, make sure to release
         entry.release();
+        throw e;
```

Comment:
What's our definition of a 'breaking change'? Does a behavior change such as throwing an exception in a situation where it was previously swallowed count?

Reply:
I think normally this wouldn't happen unless something weird happens. In that case I think this will help us understand what the real problem is?

Reply:
And also, if start here failed, GrpcDirectStreamController will move on to `ClientCallImpl#sendMessage` which will likely fail there

Reply:
Ah, and this is all in package-protected classes. Thanks for helping to clarify.